contrib/i2pd.service

```diff
@@ -2,31 +2,59 @@
 Description=I2P Router written in C++
 Documentation=man:i2pd(1) https://i2pd.readthedocs.io/en/latest/
 After=network.target
+ConditionFileIsExecutable=/usr/sbin/i2pd
 
 [Service]
 User=i2pd
 Group=i2pd
+PermissionsStartOnly=yes
+
 RuntimeDirectory=i2pd
 RuntimeDirectoryMode=0700
 LogsDirectory=i2pd
 LogsDirectoryMode=0700
-Type=forking
-ExecStart=/usr/sbin/i2pd --conf=/etc/i2pd/i2pd.conf --tunconf=/etc/i2pd/tunnels.conf --tunnelsdir=/etc/i2pd/tunnels.conf.d --pidfile=/run/i2pd/i2pd.pid --logfile=/var/log/i2pd/i2pd.log --daemon --service
+Type=exec
```

Comment:
isn't the type called `simple` not `exec` ? or is there a difference here?

Reply:
from `man 5 systemd.service`: 
>simple proceeds with further jobs right after fork() returns, while exec will not proceed before both fork() and execve() in the service process succeeded. Note that this means systemctl start command lines for exec services will report failure when the service's binary cannot be invoked successfully (for example because the selected User= doesn't exist, or the service binary is missing).

Reply:
to me that made it seem like exec was the better choice but it probably doesn't matter too much  

i don't mind making it simple